src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalService.kt

```diff
@@ -136,6 +163,7 @@ class CalculationTransactionalService(
       }
     val sourceData = prisonService.getPrisonApiSourceData(calculationRequest.prisonerId)
     val userInput = transform(calculationRequest.calculationRequestUserInput)
+//    val calculationRequestModel = CalculationRequestModel(calculationUserInputs = userInput, calculationReason = calculationRequest.reasonForCalculation, otherReasonDescription = calculationRequest.otherReasonForCalculation)
```

Comment:
Should something happen to this?

Reply:
It was development cruft, it was removed when I sorted the checkstyles. All other changes have been made.